core/idl/wallet/tezos/tezos_like_wallet.djinni

```diff
@@ -154,9 +154,10 @@ TezosLikeOriginatedAccount = interface +c {
 }
 
 TezosConfiguration = interface +c {
-  	const TEZOS_XPUB_CURVE: string = "TEZOS_XPUB_CURVE";
+  const TEZOS_XPUB_CURVE: string = "TEZOS_XPUB_CURVE";
 	const TEZOS_PROTOCOL_UPDATE: string = "TEZOS_PROTOCOL_UPDATE";
 	const TEZOS_NODE: string = "TEZOS_NODE";
+  const TZSTATS_API_ENDPOINT: string = "TZSTATS_API_ENDPOINT";
```

Comment:
Why do we need this ? I think for comparaison you'll have to retrieve: `BLOCKCHAIN_EXPLORER_API_ENDPOINT` and compare it against default `TZSTATS_API_ENDPOINT` 🤔

Reply:
Oh you are right. Nice catch!